recaptcha_enterprise/demosite/app/templates/comment.html

```diff
@@ -15,101 +15,98 @@
-            {
-              action: "send_comment",
-            }
-        );
-        await sendComment({ token });
-      });
-    }
-  </script>
-  <!-- See: https://cloud.google.com/recaptcha-enterprise/docs/create-assessment -->
-  <script type="text/javascript">
-    async function sendComment({ comment = "", token }) {
-      // Include the token for server-side assessment.
-      // Include the user input values from your form to pass to the backend.
-      const body = {
-        comment,
-        token,
-      };
-      // Code for fetching the assessment from server-side goes here.
-      // Refer to demo app backend code for more information.
-      // If you already use a library or framework for event handlers, you 
-      // can handle events your usual way.
-      const score = await fetchServerResponse({
-        body,
-        url: "on_comment_submit",
-      });
-      // In this demo, the assessment score is displayed in the client.
-      // But generally, you should avoid using the assessment response in the
-      // client and handle it on the server-side.
-      useAssessmentInClient(score);
-    }
-  </script>
-  <!-- ATTENTION: reCAPTCHA Example (Client Part 1 of 2) Ends -->
-  <meta name="viewport" content="width=device-width,initial-scale=1" />
-  <meta
+  <head>
+    <meta charset="UTF-8" />
+    <title>Comment: reCAPTCHA Demo</title>
+    <!-- ATTENTION: reCAPTCHA Example (Client Part 1 of 2) Starts -->
+    <!-- https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#user-action -->
+    <script src="https://www.google.com/recaptcha/enterprise.js?render={{site_key}}"></script>
+    <script type="text/javascript">
+      // If using a library or framework, can handle events your usual way
+      function handleClick(event) {
+        event.preventDefault();
+        grecaptcha.enterprise.ready(async () => {
+          const token = await grecaptcha.enterprise.execute("{{site_key}}", {
+            action: "send_comment",
+          });
+          await sendComment({ token });
+        });
+      }
+    </script>
+    <!-- https://cloud.google.com/recaptcha-enterprise/docs/create-assessment -->
+    <script type="text/javascript">
+      async function sendComment({ comment = "", token }) {
+        // Include the token for server-side assessment
+        // Include the user input values from your form
+        const body = {
+          comment,
+          token,
+        };
+        // Fetch from your backend; assessments must be server-side!
+        // Refer to the backend demo code for more information
+        // If using a library or framework, can fetch your usual way
+        const score = await fetchServerResponse({
+          body,
+          url: "on_comment_submit",
+        });
+        // DON'T include an assessment your real response
+        // This demo uses a score in client-side code only to illustrate
+        // It's the most secure to handle things server-side!
```

Comment:
```suggestion
        // In this demo, the assessment score is displayed in the client.
       // But generally, you should avoid using the assessment response in the
       // client and handle it on the server-side.
```

Either the above comment or feel free to combine them as needed.